lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -50,26 +53,48 @@ def _setup(
         self.bad_date_payload = bad_date_payload
         self.error_payload = error_payload
         self.empty_es_response_payload = empty_es_response_payload
+        self.index_prefix = index_prefix
+        self.index_version = index_version
```

Comment:
Yeah I can totally remove these 2 since they no longer are useful as we extracting index from the metadata and all other APIs currently only use `run-data` prefix.